### DISCUSSION TITLE
Loop Optimization à la Loopy

### DISCUSSION THREAD

**Discussion by ratnania** [pyccel/pyccel#1045] [Ideas]
Add code transformations capabilities to Pyccel, à la Loopy
https://documen.tician.de/loopy/tutorial.html

The code transformation will be done only after the semantic passes.
It is therefor interesting to have a **py2py** function that allows us to tranform our Python code and keep working on a Python code, until our transformations are ok.

**Comment by EmilyBourne** [comment]
> It is therefor interesting to have a py2py function that allows us to transform our Python code and keep working on a Python code, until our transformations are ok.

This exists with the command
`pyccel my_file.py --language=python --output=__pyccel__`
